### PR TITLE
SAK-42018: Remove duplicate button and translate button

### PR DESCRIPTION
**Pull request by alejandromf**
Remove duplicate button and translate button.

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js [hunk 1]
```diff
@@ -367,16 +366,18 @@ jQuery.fn.threadsSorter = function() {
 			if(flip %2 == 0){
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js]: These two blocks of code could be put in a single function that takes a boolean.

https://gist.github.com/adrianfish/3842cb1205950f3386a77fad93c1b949

Comment (alejandromf) [reply msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js]: Thanks for the suggestion, works correctly

File: msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js [hunk 2]
```diff
@@ -367,16 +366,18 @@ jQuery.fn.threadsSorter = function() {
...
 			}
 			else {
 				$($this[0].tBodies[0].rows).not(".hierItemBlock").show();
 				//Sync icons showing consitent for expanding 
-				imageObj.attr({'src': imageCollapseExpandUrl, 'alt':'Expand/Collapse', 'title':'Expand/Collapse'});
-				$("tr.hierItemBlock td:first-child img").attr({'src': imageExpandUrl, 'alt':'Expand/Collapse', 'title':'Expand/Collapse'});
+				imageObj.attr({'src': imageCollapseExpandUrl});
+				$("tr.hierItemBlock td:first-child img").attr({'src': imageExpandUrl});
+			}
+			if($('iframe.portletMainIframe',parent.document).length>0){
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js]: Can you add spaces length > 0 ? Makes it a bit clearer.

File: msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js [hunk 3]
```diff
@@ -357,24 +357,19 @@ jQuery.fn.threadsSorter = function() {
...
-			else {
-				$($this[0].tBodies[0].rows).not(".hierItemBlock").show();
-				//Sync icons showing consitent for expanding 
-				imageObj.attr({'src': imageCollapseExpandUrl});
-				$("tr.hierItemBlock td:first-child img").attr({'src': imageExpandUrl});
-			}
+			var toggleThreadSorter = function (el, show) {
+				  $(el.tBodies[0].rows).not(".hierItemBlock").toggle(show); 
+				  imageObj.attr({'src': show ? imageExpandCollapseUrl : imageCollapseExpandUrl});
+				  $("tr.hierItemBlock td:first-child img").attr({'src': show ? imageCollapseUrl : imageExpandUrl})
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/js/forumTopicThreadsSorter.js]: Indentation looks a bit wrong here. All small stuff but just improves readability.

**Pull request merged**